addons/website_event_booth_sale/views/event_booth_templates.xml

```diff
@@ -3,13 +3,20 @@
 
     <template id="event_booth_registration" inherit_id="website_event_booth.event_booth_registration">
         <xpath expr="//h5[@name='booth_category_name']" position="after">
-            <span t-if="booth_category.price" class="font-weight-normal text-muted" t-field="booth_category.price_reduce"
-                  t-options="{'widget': 'monetary', 'display_currency': website.pricelist_id.currency_id}"
-                  groups="account.group_show_line_subtotals_tax_excluded"/>
+            <t t-if="booth_category.price">
```

Comment:
AFAIK, in the global e-commerce logic, we should only display the reduction if the pricelist is configured to show the discount to the user, why isn't it respected in website_event_booth_sale ? :thinking: